enterprise/cli/licenses.go

```diff
@@ -0,0 +1,111 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+
+	agpl "github.com/coder/coder/cli"
+	"github.com/coder/coder/cli/cliui"
+	"github.com/coder/coder/codersdk"
+)
+
+var jwtRegexp = regexp.MustCompile(`^[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+$`)
+
+func licenses() *cobra.Command {
+	cmd := &cobra.Command{
+		Short:   "Add, remove, and list licenses",
+		Use:     "licenses",
+		Aliases: []string{"license"},
+	}
+	cmd.AddCommand(
+		licenseAdd(),
+	)
+	return cmd
+}
+
+func licenseAdd() *cobra.Command {
+	var (
+		filename string
+		license  string
+		debug    bool
+	)
+	cmd := &cobra.Command{
+		Use:  "add",
```

Comment:
Suggestions:
- Consider adding a `Short`, something like: "Add a license file or string to deployment"
- Consider adding one example each for `--file(name)` and `--license`)

Since `add` takes no arguments, adding the examples would be helpful (I imagine).